js/JS-createElement/js-createelement_toast-messages/js/index.js

```diff
@@ -6,8 +6,25 @@ const clearButton = document.querySelector('[data-js="clear-button"]');
 
 addButton.addEventListener("click", () => {
   // Exercise: Append a new entry to the toast messages container
+
+  // Generate new <li></li> element
+  let newToastEntry = document.createElement("li");
```

Comment:
You can use 'const' to declare this var, bc it will not be reassigned later:
https://stackoverflow.com/questions/72138238/const-let-or-none-for-addeventlistener-what-would-be-correct

Reply:
Thanks, @LuamB :)